README.md

```diff
@@ -313,8 +334,29 @@ If you wish to build it anyway, install Python and py2exe, and then simply run `
 * **`devscripts/update-version.py [revision]`** - Update the version number based on current date
 * **`devscripts/set-variant.py variant [-M update_message]`** - Set the build variant of the executable
 * **`devscripts/make_lazy_extractors.py`** - Create lazy extractors. Running this before building the binaries (any variant) will improve their startup performance. Set the environment variable `YTDLP_NO_LAZY_EXTRACTORS=1` if you wish to forcefully disable lazy extractor loading.
+* **`devscripts/make_changelog.py`** - Create a markdown changelog using short commit messages. It can also update the `CONTRIBUTORS` file to add new contributors from these commits.
+For more info see `make_changelog.py --help`.
 
-You can also fork the project on GitHub and run your fork's [build workflow](.github/workflows/build.yml) to automatically build a full release
+### Forking the project
+If you fork the project on GitHub, you can run your fork's [build workflow](.github/workflows/build.yml) to automatically build the selected version(s) as artifacts. Alternatively, you can run the [release workflow](.github/workflows/release.yml) or enable the [nightly workflow](.github/workflows/release-nightly.yml) to create full (pre-)releases. These can be configured in the following way:
+
+```yml
+vars.PUSH_VERSION_COMMIT: Push a version update commit to master during release workflow
+vars.BUILD_NIGHTLY: Create a nightly release on every push to master branch
+secrets.GPG_SIGNING_KEY: Private GPG key used for signing SHA files
+
+# Both need to be set to push build to archive repo as release:
+vars.ARCHIVE_REPO: Repository for archiving pre-releases, e.g. `yt-dlp/yt-dlp-nightly-builds`
+secrets.ARCHIVE_REPO_TOKEN: Personal Access Token with contents:write permission for archive repo
+
+# Publish to PyPI
+secrets.PYPI_TOKEN: Token for PyPI
+
+# Push to Homebrew taps repository (requires publish to PyPI)
+secrets.BREW_TOKEN: Private deploy key for Homebrew taps repo
+# Note: the brew workflow step and `update-formulae.py` are tailored specifically to yt-dlp
+```
+Additionally, you may want to add a channel for your repository in `yt_dlp.update.UPDATE_SOURCES` to make `--update-to`/`-U` compatible with your fork.
```

Comment:
These does not need to be documented her imo. It can go in wiki or as comment in workflow file. `BUILD_NIGHTLY` is the only option that remotely could be useful to most people